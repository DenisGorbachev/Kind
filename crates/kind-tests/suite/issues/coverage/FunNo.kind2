#axiom
Name : Type

#axiom
Teste : Name

Char.is_whitespace (c: Name) : U60
Char.is_whitespace Teste = _