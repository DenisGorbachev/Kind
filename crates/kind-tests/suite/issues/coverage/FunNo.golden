   WARN  This function does not cover all the possibilities!

      * Missing case : _ 

      /--[suite/issues/coverage/FunNo.kind2:7:1]
      |
    6 |    
    7 |    Char.is_whitespace (c: Name) : U60
      |    v-----------------
      |    \Here!
    8 |    Char.is_whitespace Teste = _


